### DISCUSSION TITLE
LAN IP on Mac returns N/a despite script working as expected

### DISCUSSION THREAD

**Discussion by Normanras** [erikw/tmux-powerline#413] [Q&A] [answered]
So this is odd. I have the WAN IP working as expected but the LAN IP script is just returning `N/a`. What makes this especially odd is that I grabbed the `lan_ip.sh` script and pulled it apart separately from the plugin. It all works as expected. Each line does the correct thing and when run together (including the `shell_is` references), I get my LAN IP... 

To add to the weirdness, if I strip down the script within the plugin to just `echo` my IP address, it still returns n/a.

Happy to debug this further, but could use some direction. Not sure where tmux isn't happy and isn't keeping the values. 

EDIT: My only other thought is that tmux doesn't have access to `ifconfig` commands? I'm running `ifstat` as well without issue, but since I'm on a mac, both `ifstat` and `tmux` are installed via brew were as `ifconfig` is built into the kernel. Also, I've made sure the permissions in the Powerline segments folder has the script as executable.

**Comment by aloyr** [comment, marked as answer]
Hi @Normanras , I had the same thing happen to my setup and I fixed mine by adding _/sbin_ to the _PATH_ environment variable inside _~/.tmux.conf_

here is the full line:
`set-environment -g PATH "/opt/homebrew/sbin:/opt/homebrew/bin:/opt/local/sbin:/opt/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"`
I hope that helps you too.

**Comment by xx4h** [comment]
Hi @Normanras and @aloyr,

did you have the `set-environment` for `PATH` in use before and just added `/sbin` to it?
Which macOS are you on, as I am not able to reproduce that on `Ventura 13.6.6`.
